tools/manifest/sourcefile.py

```diff
@@ -304,7 +302,7 @@ def url(self):
 
     @cached_property
     def hash(self):
-        # type: () -> bytes
+        # type: () -> Text
```

Comment:
I kind of feel like this ought to be bytes, but I don't know what the impact of changing the `encode` in Py3 to a `decode` in Py2 is.

Reply:
Right, I agree with you. I'll try to look for a different solution.